packages/account-v2/src/utils/formattedLoginHistoryData.ts

```diff
@@ -0,0 +1,28 @@
+import dayjs from 'dayjs';
```

Comment:
This seems like a good alternative to moment, we can opt for this library instead of moment in other places too.
This library is not added to the package.json dependencies of account-v2 ?

Reply:
its currently present in package-lock.json file 

![Screenshot 2024-03-27 at 11 28 49 AM](https://github.com/binary-com/deriv-app/assets/158162395/8ca87db7-39db-42a2-9d0a-d49f39b5a4a0)

do i have to make any change to this ?

Reply:
Since you are using this package in account-v2, it should be present in the account-v2 package.json file.